scripts/lib/CIME/case.py

```diff
@@ -957,6 +952,12 @@ def configure(self, compset_name, grid_name, machine_name=None,
 
         env_batch.set_batch_system(batch, batch_system_type=batch_system_type)
         env_batch.create_job_groups(bjobs)
+
+        if walltime:
```

Comment:
Important change. Limit user requests to specific job groups that only exist after create_job_groups.